docs/source/_ext/opnsense_client_api.py

```
@@ -0,0 +1,122 @@
+    Args:
+        mixin_class (type[Any]): Mixin class to inspect.
+
+    Returns:
+        list[str]: Ordered list of public method names defined on the mixin.
+    """
+    public_methods: list[str] = []
+    for name, value in mixin_class.__dict__.items():
+        if name.startswith("_"):
+            continue
+        if isinstance(value, (staticmethod, classmethod)):
+            value = value.__func__
```

Ruff (0.15.9)

[warning] `for` loop variable `value` overwritten by assignment target

(PLW2901)